README.md

```diff
@@ -23,51 +21,12 @@ ember install ember-a11y-testing
 
 ## Usage
 
-### Testing Usage
-By default, Ember A11y Testing will automatically begin running during _acceptance_ tests. It
-also injects the `axe` object globally during development so you can run tests
-while developing your application as well.
-
-_Note:_ any tests run with Ember A11y Testing will adjust the testing container
-to occupy the entire screen. This is to simulate the actual application
-environment, as browsers adjust styles at small sizes for accessibility reasons.
-It will reset itself at the conclusion of testing though.
-
-#### Disabling/Enabling Axe During Tests
-
-By default, the axe-core tests only run during acceptance tests. In order to
-enable them for other tests, simply run the following at the beginning of your
-testing module:
-
-```javascript
-axe.ember.turnAxeOn();
-```
-
-On the flip side, if you want to turn tests off, simply use:
-
-```javascript
-axe.ember.turnAxeOff();
-```
+Ember A11y Testing injects the `axe` global during development and tests so that
+you can run accessibility audits while developing your application. It also
+provides some Ember-specific tools so that you can integrate accessibility
+checks into your workflow easily.
```

Comment:
We can update this in the future, but I feel like the examples further down should suffice.